ansible/yarn.service

```
@@ -0,0 +1,9 @@
+[Unit]
+Description=Yarn service
+[Service]
+User=vagrant
+Group=vagrant
+TimeoutStartSec=900
+WorkingDirectory=/vagrant/zapisy/
+ExecStart=/usr/local/bin/yarn
+Type=simple
```

To nie jest usługa systemowa (która ma działać wiecznie), tylko _zadanie_, które trzeba wykonać raz i kropka.
```suggestion
Type=oneshot
```
https://www.freedesktop.org/software/systemd/man/systemd.service.html